pyramid_okta/security/authentication.py

```diff
@@ -65,6 +64,7 @@ def effective_principals(self, request):
             effective_principals.append(user_id)
         else:
             effective_principals.append(Authenticated)
+            effective_principals.append('login_required')
```

Comment:
yes its made up :) so are many things with pyramid and I like that flexibility. We don't need it to be like this. Can rename it to group name or a better permission ACL.